### PR TITLE
Version 2.0.0.rc1 release

### PR DESCRIPTION
**Pull request by asif-docusign**
### Changed
- Added support for version v1.1.0-1.0.4 of the DocuSign WebForms API.
- Updated the SDK release version.

## Important Action Required
- User needs to update the base URL in your codebase (if passing to SDK explicitly) and remove the /v1.1 component at the end of the URL for the SDK to work properly with this release.

### CODE REVIEW

File: tests/docs/private.pem [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (garg-mudit) [inline tests/docs/private.pem:1]: Lets put this in gitignore for the external repo.

**Review by garg-mudit — CHANGES_REQUESTED** [review]
Minor Change request.

File: tests/docs/private.pem (hunk 1 above)
Comment (asif-docusign) [reply tests/docs/private.pem:1]: Done

**Comment by asif-docusign** [comment]
@garg-mudit Done the requested change. Please review.

**Review by garg-mudit — APPROVED** [review]
LGTM

**Pull request merged**